apps/etterlatte-migrering/src/main/kotlin/migrering/MigreringRiver.kt

```diff
@@ -1,34 +1,42 @@
 package no.nav.etterlatte.migrering
 
 import kotlinx.coroutines.runBlocking
-import no.nav.etterlatte.libs.common.rapidsandrivers.eventName
-import no.nav.etterlatte.migrering.pen.PenKlient
+import no.nav.etterlatte.libs.common.rapidsandrivers.EVENT_NAME_KEY
 import no.nav.etterlatte.rapidsandrivers.migrering.Migreringshendelser.MIGRER_SPESIFIKK_SAK
 import no.nav.etterlatte.rapidsandrivers.migrering.Migreringshendelser.START_MIGRERING
 import no.nav.helse.rapids_rivers.JsonMessage
 import no.nav.helse.rapids_rivers.MessageContext
 import no.nav.helse.rapids_rivers.RapidsConnection
 import org.slf4j.LoggerFactory
+import rapidsandrivers.SAK_ID_FLERE_KEY
+import rapidsandrivers.SAK_ID_KEY
 import rapidsandrivers.migrering.ListenerMedLoggingOgFeilhaandtering
-import rapidsandrivers.sakId
+import rapidsandrivers.sakIdFlere
 
-internal class MigreringRiver(rapidsConnection: RapidsConnection, private val penKlient: PenKlient) :
+internal class MigreringRiver(rapidsConnection: RapidsConnection) :
     ListenerMedLoggingOgFeilhaandtering(START_MIGRERING) {
     private val logger = LoggerFactory.getLogger(this::class.java)
 
     init {
-        initialiserRiver(rapidsConnection, hendelsestype)
+        initialiserRiver(rapidsConnection, hendelsestype) {
+            validate { it.requireKey(SAK_ID_FLERE_KEY) }
```

Comment:
En ting jeg kom på nå, er at vi er nødt til å håndtere to "ulike" lister av sakId-er. 
Vi kommer til å kjøre separate migreringer for brukere som har en løpende ytelse på reformtidspunkt (1.1.2024), og brukere hvor ytelsen gjenoppstår siden de nå kan få til 20 år. 

Jeg tror vi bør ta høyde for dette i denne riveren. F. eks med en egen boolean key `LOEPENDE_JANUAR_2024` e.l. som vi kan benytte til å angi hva slags saker det er man sender inn. Dette parameteret bør også sendes med videre til `MIGRER_SPESIFIKK_SAK` riveren, og til slutt bli med som parameter i `penKlient.hentSak`, som krever at vi angir dette query parameteret.